SUB-PROPERTY PERMUTATIONS:.md

```diff
@@ -0,0 +1,243 @@
+SUB-PROPERTY PERMUTATIONS:
+
+Axiom: for [B] object_subProperty of [A], any {x,y} pair of instances in B will occur in [A].
```

Comment:
I would try to make the axiom more explicit: for a sub-property relationship B as owl:subPropertyOf A, any pair {x, y} related by B must also be related by A